ee/clickhouse/queries/trends/person.py

```diff
@@ -74,19 +80,20 @@ def get_query(self) -> Tuple[str, Dict]:
             filter=self.filter,
             team_id=self.team.pk,
             entity=self.entity,
-            should_join_distinct_ids=True,
-            should_join_persons=True,
-            extra_fields=["distinct_id", "team_id"],
-            extra_person_fields=["created_at", "person_props", "is_identified"],
+            should_join_distinct_ids=False if self.is_aggregating_by_groups else True,
+            should_join_persons=False if self.is_aggregating_by_groups else True,
+            extra_fields=[] if self.is_aggregating_by_groups else ["distinct_id", "team_id"],
```

Comment:
Realized this could just be a single query after we agreed we can just pull data from postgres with the ids. There's very few extra params needed when it's a group query besides these 3. No need to push an extra parameters down to columnoptimizer or the eventquery builder